packages/cli/src/options/beaconNodeOptions/network.ts

```diff
@@ -16,6 +16,7 @@ export interface INetworkArgs {
   "network.blockCountTotalLimit": number;
   "network.blockCountPeerLimit": number;
   "network.rateTrackerTimeoutMs": number;
+  "network.passGossipAttestationsToForkchoice": boolean;
```

Comment:
I think we should make the flag in reverse such that the default is false. Change to
```
dontSendGossipAttestationsToForkchoice
```